frontend/packages/console-plugin-sdk/src/typings/nav-items.ts

```diff
@@ -29,62 +22,74 @@ namespace ExtensionProperties {
     insertAfter?: string | string[];
   }
 
+  /** @deprecated Use dynamic plugin nav extensions instead. */
```

Comment:
Yeah, we have a separate story for deleting these files, but I had to update the types since they were referencing components that I changed. I didn't want to make updates here without also clarifying that these extensions are not actually supported.

Reply:
In reality, these changes might not even end up in master because @zherman0 is working on the story to delete this. We could merge that first, which would make these changes obsolete.